cms/djangoapps/contentstore/views/item.py

```diff
@@ -595,6 +595,13 @@ def _duplicate_item(parent_usage_key, duplicate_source_usage_key, user, display_
             else:
                 duplicate_metadata['display_name'] = _("Duplicate of '{0}'").format(source_item.display_name)
 
+        asides_to_create = []
+        for aside in source_item.runtime.get_asides(source_item):
+            for field in aside.fields.values():
+                if field.scope in (Scope.settings, Scope.content,) and field.is_set_on(aside):
```

Comment:
@cpennington I've updated the PR and added the block to remove all fields in aside that are not  `Scope.content` and `Scope.settings` before `create_item` statement.